### DISCUSSION TITLE
Supported file types on ERC-721 Metadata page

### DISCUSSION THREAD

**Discussion by jessepinkman9900** [blockscout/blockscout#6022] [Q&A] [answered]
- Blockscout Version: `4.1.6` 
- I can see that `.jpg` and `.png` are supported 
- Would like to know if any other image formats are supported, like `.svg`?

**Comment by vbaranov** [comment, marked as answer]
Yes, `svg` supported as well. Generally speaking, everything that can be used as a source to `<image>` or `<video>` tags are supported https://developer.mozilla.org/ru/docs/Web/HTML/Element/img.
